hybrasyl/Objects/User.cs

```diff
@@ -2370,6 +2573,11 @@ public void AssailAttack(Direction direction, Creature target = null)
                         {
                             target = user;
                         }
+                        var npc = obj as Merchant;
+                        if (npc != null)
+                        {
+                            target = npc;
+                        }

```

Comment:
Move the declaration outside as `VisibleObject obj` and check for null after the case statement is finished?